f3rm_robot/field_adapter.py

```diff
@@ -0,0 +1,72 @@
+from typing import Dict
+
+import torch
+import torch.nn as nn
+from jaxtyping import Float
```

Comment:
uhm, why are we using the jaxtyping Float?

Reply:
That's what nerfstudio latest uses. Seems like it is better alternative to torchtyping